aspnet/getting-started/installing-on-linux.rst

```diff
@@ -145,7 +145,41 @@ To build libuv you should do the following::
 Using Docker
 ------------
 
-Instructions on how to use the ASP.NET 5 Docker image can be found here: http://blogs.msdn.com/b/webdev/archive/2015/01/14/running-asp-net-5-applications-in-linux-containers-with-docker.aspx
+The following instructions were tested with Docker 1.8.3 and Ubuntu 14.04.
+
+Install Docker
+^^^^^^^^^^^^^^
+
+Instructions on how to install Docker can be found in the `Docker Documentation <https://docs.docker.com/installation/>`_.
+
+Create a Container
+^^^^^^^^^^^^^^^^^^
+
+Inside your application folder, you create a ``Dockerfile`` which should looks something like this::
+
+    # Base of your container
+    FROM microsoft/aspnet:latest
+
+    # Copy the project into folder and then restore packages
+    COPY project.json /app/
```

Comment:
can we stop doing this? Just do copy . /app and then remove the other line that does that.

We still have a bunch of old samples doing this but it breaks if the app you are uploading has already been restored on the host as the host lock file replaces the one that was restored inside the container. This can break your app if you are using a package that relies on OS specific implementation packages.